components/files/docs.json

```diff
@@ -0,0 +1,7 @@
+{
+  "component": "file",
```

Comment:
This should be named `files`, since there is both a `files` and `file` component involved